specs/validator/0_beacon-chain-validator.md

```diff
@@ -262,12 +270,12 @@ def get_eth1_vote(state: BeaconState, previous_eth1_distance: uint64) -> Eth1Dat
 
 ##### Signature
 
-Set `header.signature = block_signature` where `block_signature` is obtained from:
+Set `signed_header.signature = block_signature` where `block_signature` is obtained from:
```

Comment:
`header` or `signed_header` isn't previously defined, nor is it used subsequently. A valid signature here also depends on already creating the block body.

I suggest that we remove `Signature` section ehre and create a `Signed Beacon Block` section after `Block Body`. This section will construct the wrapper container and add in the signature. It will also include a note about how this is the message that is broadcast to the network

Reply:
(This issue was generally here before you made edits. We should fix it none the less)